etna/search/tests/test_forms.py

```diff
@@ -41,116 +41,60 @@ def test_empty_form(self):
         form.is_valid()
 
         self.assertEqual(
-            form.selected_filters(),
-            {
-                "levels": [],
-                "topics": [],
-                "collections": [],
-                "closure_statuses": [],
-                "catalogue_sources": [],
-            },
-        )
-
-    def test_selected_group_is_excluded(self):
-        form = CatalogueSearchForm(
-            {
-                "group": "group:tna",
-            }
-        )
-
-        form.is_valid()
-
-        self.assertEqual(
-            form.selected_filters(),
-            {
-                "levels": [],
-                "topics": [],
-                "collections": [],
-                "closure_statuses": [],
-                "catalogue_sources": [],
-            },
+            form.selected_filters,
+            {},
         )
 
-    def test_selected_level(self):
+    def test_with_recognised_filter_values(self):
         form = CatalogueSearchForm(
             {
                 "group": "group:tna",
-                "levels": ["level:level-one"],
+                "level": ["Division"],
+                "collection": ["WO"],
             }
         )
 
-        form.is_valid()
+        self.assertTrue(form.is_valid())
 
         self.assertEqual(
-            form.selected_filters(),
+            form.selected_filters,
             {
-                "levels": ["level:level-one"],
-                "topics": [],
-                "collections": [],
-                "closure_statuses": [],
-                "catalogue_sources": [],
+                "collection": [
+                    (
+                        "WO",
+                        "WO - War Office, Armed Forces, Judge Advocate General, and related bodies",
+                    )
+                ],
+                "level": [("Division", "Division")],
             },
         )
 
-    def test_selected_topic(self):
+    def test_with_non_recognised_filter_values(self):
         form = CatalogueSearchForm(
             {
                 "group": "group:tna",
-                "topics": ["topic:topic-one"],
+                "level": ["Division"],
+                "collection": ["WO"],
```

Comment:
Why are we putting recognised values and checking for level, collection?

Reply:
Because it makes for a more complete test... I'm interested in what happens when `form.cleaned_data` contains some values, but not others